src/atoms/Type/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-export const weights = [300, 400, 500, 700, 900]
+export const weights = [300, 400, 500, 700, 900, '300', '400', '500', '700', '900']
```

Comment:
What's the reasoning behind having both int and string versions of weights? 💭

Reply:
Ahh yes, so this is just to allow for both strings and ints to be passed in as props. I have two reasons for this: 
1. flexibility
2. because the knobs addon doesn't like number values as well as options knob (kept erring on me)

Reply:
Ahh ok. That makes sense, but also feels a bit unmaintainable over time. Might be good to add a general utils function that handles this duplication of integers into strings? 💭

Reply:
Great suggestion! Added in latest commit! 😬